src/shared/port-finder.js

```diff
@@ -34,20 +37,16 @@ export class PortFinder {
   /**
    * Request a specific port from `hostFrame`
```

Comment:
```suggestion
   * Request a specific port from the host frame
```

`hostFrame` is no longer an argument to this method.